Arrowgene.Ddon.Database/Files/Database/Script/AddMissingOnDeleteCascades_migration.sql

```diff
@@ -0,0 +1,88 @@
+﻿PRAGMA foreign_keys=OFF;
+
+BEGIN TRANSACTION;
```

Comment:
Migrations are run in a transaction already, this would probably conflict with the already existing transaction

Reply:
It was not throwing any exceptions for me. I remember seeing a setting for nested transactions that can be turned on. Is it maybe active by default?
But good that you mentioned it, from what I understand the pragma foreign key check can only be turned off before transactions begin.. I can work around that by manually committing the transaction and do my migration.